README.md

```diff
@@ -58,6 +58,9 @@ httping -url requested_url [OPTIONS]
 
 -json
   If specified, outputs the results in json format
+
+-noProxy
```

Comment:
I am thinking what's the preferred default. I guess it's good to follow system settings and have to be explicit if we'd like to use no proxy. What's your take on this?

Reply:
I disagree. 

"System proxy" is essentially "system-wide proxy settings", and 99% of the time it's empty. 

However, when they are set, then (at least on Windows): Chrome, Firefox - will use these settings. So my reasoning is, if system proxy is bad, then the user will probably notice this as they try to use their browser. Otherwise, the user probably expects connections via proxy, especially if they use a specialized tool like Httproxy. However

That said, `-noProxy` *may* be a bad name. I thought of `-direct`, but it may sound misleading. I also thought of `-withoutProxy` or `-forceDirectConn(ection)` - but I thought that these flags are unnecessarily long. 

However, feel free to let me know what you think. Thanks!

Reply:
I think I haven't explained myself well as we're probably agreeing that the user expectations are for the tool to follow system wide settings unless explicitly requested. 

As we're also printing proxy info during runtime my worries are reduced (user getting unknowingly misleading data through httping as e.g. they could be using a proxy and not know about it)